<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="ListenerSuite" duration-ms="73" started-at="2019-03-17T10:25:53Z" finished-at="2019-03-17T10:25:53Z">
    <groups>
    </groups>
    <test name="Testing" duration-ms="73" started-at="2019-03-17T10:25:53Z" finished-at="2019-03-17T10:25:53Z">
      <class name="com.testng.listener.ListenerDemo">
        <test-method status="FAIL" signature="testDemoFailedMethod()[pri:0, instance:com.testng.listener.ListenerDemo@1fa1d9b]" name="testDemoFailedMethod" duration-ms="26" started-at="2019-03-17T10:25:53Z" finished-at="2019-03-17T10:25:53Z">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<[ABC]> but was:<[XYZ]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<[ABC]> but was:<[XYZ]>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at com.testng.listener.ListenerDemo.testDemoFailedMethod(ListenerDemo.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDemoFailedMethod -->
        <test-method status="PASS" signature="testDemoPassedMethod()[pri:0, instance:com.testng.listener.ListenerDemo@1fa1d9b]" name="testDemoPassedMethod" duration-ms="2" started-at="2019-03-17T10:25:53Z" finished-at="2019-03-17T10:25:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDemoPassedMethod -->
        <test-method status="SKIP" signature="testDemoSkippedMethod()[pri:0, instance:com.testng.listener.ListenerDemo@1fa1d9b]" name="testDemoSkippedMethod" duration-ms="2" started-at="2019-03-17T10:25:53Z" finished-at="2019-03-17T10:25:53Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Method is skipped!]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Method is skipped!
at com.testng.listener.ListenerDemo.testDemoSkippedMethod(ListenerDemo.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDemoSkippedMethod -->
      </class> <!-- com.testng.listener.ListenerDemo -->
    </test> <!-- Testing -->
  </suite> <!-- ListenerSuite -->
</testng-results>
